.github/workflows/deploy_oidc.yml

```diff
@@ -15,11 +17,29 @@ on:
 jobs:
   deploy_oidc:
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
     environment: ${{ inputs.deploy_env }}
     steps:
       - name: Check out repo
         uses: actions/checkout@v3
 
+      - name: Configure awscli (local runner)
+        if: ${{ env.ACT }} # OIDC doesn't work when using ACT, as AWS user accounts don't have access to the OIDC role by default
```

Comment:
what's ACT?

Reply:
This is automatically set when running the action locally using https://github.com/nektos/act

Reply:
ah makes sense. thanks